### DISCUSSION TITLE
如何关闭在按"shift"切换中/英时输入光标上方的“中/A”的提示图标？

### DISCUSSION THREAD

**Discussion by HerrScherr648** [rime/home#1687] [Q&A]
![image](https://github.com/user-attachments/assets/f3b236d7-739a-4ef1-91c8-12aa4f81d54b)
如图，我不需要这种提示，应该怎么关闭？

**Comment by kpcheong** [comment]
用户文件里的weasel.yaml文件这一句修改成false，deploy就可以
![image](https://github.com/user-attachments/assets/46511c9c-f1b9-4128-a7a5-6a6b1a96d2b3)

**Reply by HerrScherr648** [reply]
谢谢，不过这个在`程序文件夹`的data文件夹内
![image](https://github.com/user-attachments/assets/c1527dd3-de52-4f0c-a576-c246e773d4de)
可惜我这里没有这种注释

**Comment by MacKenia** [comment]
如何对这个图标进行自定义呢？
